core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -1349,6 +1350,12 @@ private Result toInnerStorageType(final Result result, final Type storageType) {
     return result;
   }
 
+  @Override public Result visitNamedParam(RexNamedParam namedParam) {
```

Comment:
Might be good to have a JIRA for this?

Reply:
Done: https://bodo.atlassian.net/browse/BS-532